src/connect/create-ethrpc-configuration.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+function createEthrpcConfiguration(configuration) {
+  var ethrpcConfiguration = {
+    connectionTimeout: 60000,
+    errorHandler: function (err) { if (err) console.error(err); },
+  };
+  ethrpcConfiguration.httpAddresses = configuration.httpAddresses;
+  ethrpcConfiguration.wsAddresses = configuration.wsAddresses;
+  ethrpcConfiguration.ipcAddresses = configuration.ipcAddresses;
+  ethrpcConfiguration.networkID = configuration.networkID;
```

Comment:
I think this needs to match the casing used on line 51 of src/connect/index.js (i.e. it should be `networkId`, not `networkID`): https://github.com/AugurProject/augur.js/blob/52e5c330914adb2813b2093f1a694e6772c84533/src/connect/index.js#L51